### PR TITLE
Import new content from 10002700:TYCE4AQHdGuE

### PR DESCRIPTION
**Pull request by FabulousCupcake**
This PR adds new content from `master.mdb` in version `10002700:TYCE4AQHdGuE`

### CODE REVIEW

File: src/data/conditions-desc.csv [hunk 1]
```diff
@@ -7,7 +7,7 @@
 "おっと……足元には気を付けて！\nトレーニングが失敗しやすい","Whoops! Watch your steps!\nIncreased training fail rate."
 "飲み込みの早さはピカイチ！\nあらゆる戦術のヒントを得る","Such a fast learner!\nGain discounts to all skills."
 "一緒に練習すると楽しいね！\nトレーニング相手ともっと仲良くなれる","It's fun exercising together!\nGet to know your training partner better."
-"傑物ッ！　学園を背負って立つ存在！！\n記者と理事長がもっと評価してくれる","Outstanding! The academy is proud of you!\nJournalists and the Principal will appreciate you more."
+"傑物ッ！　学園を背負って立つ存在！！\n記者や理事長などがもっと評価してくれる","Outstanding! The academy is proud of you!\nJournalists and the Principal will appreciate you more."
```
[the remark this answers is not in the document]
Comment (FabulousCupcake) [reply src/data/conditions-desc.csv:10]:
```suggestion
"傑物ッ！　学園を背負って立つ存在！！\n記者や理事長などがもっと評価してくれる","Outstanding! The academy is proud of you!\nJournalists, Directors, etc. will appreciate you more."
```

[the remark this answers is not in the document]
Comment (FabulousCupcake) [reply src/data/conditions-desc.csv:10]: Director? Chairman? Principal? I think I need to check the new scenario to find out the appropriate term here

[the remark this answers is not in the document]
Comment (robflop) [reply src/data/conditions-desc.csv:10]: I think we can keep using "Principal" here.

[the remark this answers is not in the document]
Comment (FabulousCupcake) [reply src/data/conditions-desc.csv:10]:
```suggestion
"傑物ッ！　学園を背負って立つ存在！！\n記者や理事長などがもっと評価してくれる","Outstanding! The academy is proud of you!\nJournalists and the Principals will appreciate you more."
```
Like this?

[the remark this answers is not in the document]
Comment (robflop) [reply src/data/conditions-desc.csv:10]: Yea, that seems fine.

**Review by robflop — APPROVED** [review]

**Comment by FabulousCupcake** [comment]
Thank 🙇

**Pull request merged**